### DISCUSSION TITLE
are there ways to get multiple PlanarArray types in one scene?

### DISCUSSION THREAD

**Discussion by AndreaMaestri18** [NVlabs/sionna#508] [General]
Hi everyone, I recently started working with Sionna and I was trying (with no success), to get different transmitters with different types to be active at the same time. In particular I'd like to have transmitters with different patterns in the same scene.

Do you know of any way to achieve this?

**Comment by jhoydis** [comment]
Hi @AndreaMaestri18,

This is currently not supported.
